third_party/move/evm/move-to-yul/src/abi_native_functions.rs

```diff
@@ -294,7 +294,7 @@ impl SoliditySignature {
     pub(crate) fn create_default_sig_for_encode_fun(ctx: &Context, fun: &FunctionEnv<'_>) -> Self {
         let fun_name = fun.symbol_pool().string(fun.get_name()).to_string();
         let mut para_type_lst = vec![];
-        for Parameter(para_name, move_ty) in fun.get_parameters() {
+        for Parameter(para_name, move_ty, _) in fun.get_parameters() {
```

Comment:
Not an action item for this PR, but we should ask around if this crate is being maintained and is actually usable.

If not, we might be better off getting rid of it sometime soon.